docker/hm-admin/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:6
+
+# Install google chrome, it will be use to run units and e2e tests
+RUN \
```

Comment:
For now, this Dockerfile is dev only. There is nothing prod related. So I would prefer keep it simple